### PR TITLE
Updating files for Winisim integration

### PR DESCRIPTION
**Pull request by divekarshubham**
Updating files for Winisim integration

Description
-----------

* Updating Winsim Pal
* Updating project files for Winsim

Checklist:
----------

- [ X ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: vendors/pc/boards/windows/aws_demos/config_files/ota_config.h [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+ *  This configuration parameter is sent with data requests and represents the maximum number of
+ *  data blocks the service will send in response. The maximum limit for this must be calculated
+ *  from the maximum data response limit (128 KB from service) divided by the block size.
+ *  For example if block size is set as 1 KB then the maximum number of data blocks that we can
+ *  request is 128/1 = 128 blocks. Configure this parameter to this maximum limit or lower based on
+ *  how many data blocks response is expected for each data requests.
+ *  Please note that this must be set larger than zero.
+ *
+ */
+#define otaconfigMAX_NUM_BLOCKS_REQUEST        4U
```
Comment (pvyawaha) [inline vendors/pc/boards/windows/aws_demos/config_files/ota_config.h:104]: If block size is 4KB then keep the default number of blocks requested to 1.

File: vendors/pc/boards/windows/ports/ota_pal_for_aws/ota_pal.h [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+/**
+ * @file ota_pal.h
+ * @brief Function declarations for ota_ptal.c.
```
Comment (pvyawaha) [inline vendors/pc/boards/windows/ports/ota_pal_for_aws/ota_pal.h:28]: Typo ota_pal.c

**Review by pvyawaha — CHANGES_REQUESTED** [review]

File: vendors/pc/boards/windows/aws_demos/config_files/ota_config.h (hunk 1 above)
Comment (divekarshubham) [reply vendors/pc/boards/windows/aws_demos/config_files/ota_config.h]: Done, updated in https://github.com/aws/amazon-freertos/pull/3047/commits/36bdcd94ccd9bbac42b5833848f64df46c08a026

File: vendors/pc/boards/windows/ports/ota_pal_for_aws/ota_pal.h (hunk 2 above)
Comment (divekarshubham) [reply vendors/pc/boards/windows/ports/ota_pal_for_aws/ota_pal.h]: Thanks, fixed in https://github.com/aws/amazon-freertos/pull/3047/commits/36bdcd94ccd9bbac42b5833848f64df46c08a026

**Review by pvyawaha — APPROVED** [review]

**Pull request merged**